tests/test_model.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import pytest
+import peewee
+
+from tracboat.gitlab import model
+
+
+@pytest.mark.parametrize('version', [
+    '8.4',
+    '8.5',
+    '8.7',
+    '8.13',
+    '8.15',
+    '8.16',
+    '8.17',
+    '9.0.0'
```

Comment:
p.s.: opened issue #19 for this, I think it should be addressed in another pull request since it looks like a totally different aspect than providing custom user attributes.